cmake/SetupCompilerOptions.cmake

```diff
@@ -111,6 +111,17 @@ if(BLT_DEFINES)
     message(STATUS "Added \"${BLT_DEFINES}\" to definitions")
 endif()
 
+if(COMPILER_FAMILY_IS_MSVC)
+    # Mark 32 bit executables large address aware so they can use > 2GB address space
+    if(CMAKE_SIZEOF_VOID_P MATCHES 4)
+        SET(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /LARGEADDRESSAWARE")
+    endif()
+
```

Comment:
Still not sure if we need these fixes, adding some refs to the discussion:

https://stackoverflow.com/questions/2288728/drawbacks-of-using-largeaddressaware-for-32-bit-windows-executables